packages/article-converter/src/plugins/SpanPlugin.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { attributesToProps, domToReact } from 'html-react-parser';
+import { PluginType } from './types';
+export const SpanPlugin: PluginType = (node, opts) => {
+  if (node.attribs['data-size'] === 'large') {
+    const props = attributesToProps(node.attribs);
+    return (
+      <span {...props} data-size={undefined} className={`${node.attribs.class ?? ''} u-large-body-text`}>
```

Comment:
Vil denne ta høgde for språkmarkering av ord?

Reply:
Kan ikke se for meg noe annet. Språkmarkering innebærer jo bare at man legger på `lang="en"`. Her vil `lang`-attributtet legges på med `{...props}`.